core/codenvy-hosted-platform-api-impl/src/main/java/com/codenvy/api/workspace/LimitsCheckingWorkspaceManager.java

```diff
@@ -111,9 +115,15 @@ public WorkspaceImpl startWorkspace(String workspaceId,
                                                                            ServerException,
                                                                            ConflictException {
         final WorkspaceImpl workspace = getWorkspace(workspaceId);
+        String userId;
+        try {
+            userId = userManager.getByName(workspace.getNamespace()).getId();
+        } catch (NotFoundException e) {
+            userId = getCurrentUserId();
```

Comment:
Workspace start behaviour must be strict and always predictable(the same in this case), i think that you should prohibit workspace start if there is no user owning the workspace found @sleshchenko, @skabashnyuk can you please take a look